go.mod

```diff
@@ -19,17 +19,17 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/go-github/v45 v45.2.0
 	github.com/google/gofuzz v1.2.0
-	github.com/mattn/go-runewidth v0.0.13 // indirect
-	github.com/onsi/ginkgo/v2 v2.2.0
-	github.com/onsi/gomega v1.20.1
+	github.com/mattn/go-runewidth v0.0.14 // indirect
+	github.com/onsi/ginkgo/v2 v2.4.0
+	github.com/onsi/gomega v1.22.1
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.5.0
+	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.12.0
+	github.com/spf13/viper v1.13.0
```

Comment:
@Cecile what do you think about this PR? (only the second commit)

It should come down to only bumping cobra & viper a minor version

Reply:
Release notes of both look good to me